eng/pipelines/templates/jobs/test.yml

```diff
@@ -36,9 +36,6 @@ jobs:
       ServiceDirectory: ${{ parameters.ServiceDirectory }}
       PackageInfoDirectory: $(Build.ArtifactStagingDirectory)/PackageInfo
 
-  - ${{ if eq(parameters.TestProxy, true) }}:
```

Comment:
If it updates `PATH` is still might be worth it. Just means everything is ready to go and there's no guess work since, without DEBUG tracing, it's all a silent install inside `cargo`. But if not...which begs the question "what's the point of the step, then", we can remove it.

Reply:
It absolutely prepends the path so let's leave this for now as agreed offline.